README.md

```diff
@@ -7,7 +7,7 @@
 [![docs](https://docs.rs/fuel-indexer/badge.svg)](https://docs.rs/fuel-indexer/)
 [![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
 
-The Fuel Indexer is a standalone binary that can be used to index various components of [the Fuel protocol](https://github.com/FuelLabs/fuel-specs/tree/master/specs/protocol). These indexable components include blocks, transactions, and [receipts](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md) and state within a fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
+The Fuel Indexer is a standalone binary that can be used to index various components of the blockchain. These indexable components include blocks, transactions, and receipts and state within a Fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
```

Comment:
> These indexable components include blocks, transactions, and receipts and state within a Fuel network...

Should that first 'and' be removed?

> These indexable components include blocks, transactions, receipts, and state within a Fuel network...

Reply:
That sir....is a good eye 👁️ . Change incoming.